pkgs/applications/version-management/git-and-tools/git-quick-stats/default.nix

```
@@ -0,0 +1,42 @@
+{ stdenv, fetchFromGitHub, makeWrapper, coreutils, git, gnugrep, gnused }:
+
+stdenv.mkDerivation rec {
+    pname = "git-quick-stats";
+    version = "2.0.8";
+
+    src = fetchFromGitHub {
+        owner = "arzzen";
+        repo = pname;
+        rev = "c11bce17bdb1c7f1272c556605b48770646bf807";
```

This will make future updates easier:

```suggestion
        rev = version;
```

---

Oh, missed existing tag, thanks!

---

Does not seem to be resolved